### ISSUE TITLE
Replicate tabs across computers

### ISSUE THREAD

**Issue by peytonstafford** [brillout/timer-tab#5]
I move between a Mac and a Chromebook several times, every day, according to the apps I am using. I would like to be able to start a stopwatch on one computer, and then have it appear on the other.

**Comment by brillout** [comment]
I agree that'd be super nice. It just costs a lot of dev time to achieve.

I wonder how many people need that.

And I wonder, did you find a way to achieve what you want? Is there a timer app out there that does that?

**Comment by peytonstafford** [comment]
There is the Flexible Timers app from Xintech for Mac OS and iOS. It will
sync. I just prefer being able to reset the time with one click, like your
app does.

Peyton Stafford | Voice +1 503.720.7149 | www.peytonstafford.com |  Order
my forthcoming book from ALA Editions: No Shelf Required 3: The New Era for
E-Books and Digital Content
<https://www.alastore.ala.org/content/no-shelf-required-3-new-era-e-books-and-digital-content>
.
Currently Reading
<https://www.goodreads.com/user/email_signature_destination/1026584?utm_medium=reading_link&utm_source=email_signature>
[image: Book Cover]
<https://www.goodreads.com/user/email_signature_destination/1026584?utm_medium=cover&utm_source=email_signature>

**Comment by brillout** [comment]
Maybe they are wiling to add a one-click-reset feature? That'd be faster to implement than implementing syncing :)

**Comment by peytonstafford** [comment]
That's an idea, but their system is a very simple work tracker. It records
data and saves it, so if your work is divided into only a few projects, it
can be adequate. For myself, I have several major areas of work with
multiple projects in each of them, so I need a more robust system that will
let me analyze time spent vs. money earned. I find Google Forms works well
for me. By promptly recording my work, after timing it with Timer-Tab, I
build an understandable record of how I am spending my time. I will stay
with Timer-Tab and wish you the best.

Peyton Stafford | Voice +1 503.720.7149 | www.peytonstafford.com |  Order
my forthcoming book from ALA Editions: No Shelf Required 3: The New Era for
E-Books and Digital Content
<https://www.alastore.ala.org/content/no-shelf-required-3-new-era-e-books-and-digital-content>
.
Currently Reading
<https://www.goodreads.com/user/email_signature_destination/1026584?utm_medium=reading_link&utm_source=email_signature>
[image: Book Cover]
<https://www.goodreads.com/user/email_signature_destination/1026584?utm_medium=cover&utm_source=email_signature>

**Comment by brillout** [comment]
I see and I'm more than eager to implement syncing if more people ask for it.

Rom :)
